packages/libs/client/package.json

```diff
@@ -33,7 +33,8 @@
     "pactjs:generate:contract": "pactjs contract-generate --file contracts/coin.contract.pact",
     "pactjs:retrieve:contract": "pactjs retrieve-contract --out contracts/coin.contract.pact --module coin",
     "start": "ts-node --transpile-only src/index.ts",
-    "test": "jest"
+    "test": "jest",
+    "test:integration": "jest --silent -c ./jest.integration.config.js"
```

Comment:
Why go silent?

Reply:
Primarily to not output all the console logging done by Kadena Client.  It makes the logs generated by Jest very hard to read.